xml/System/Array.xml

```diff
@@ -9534,7 +9551,15 @@ This property implements the <xref:System.Collections.ICollection?displayPropert
       <Docs>
         <summary>Gets a value that indicates whether access to the <see cref="T:System.Array" /> is synchronized (thread safe).</summary>
         <value>true if access to the <see cref="T:System.Array" /> is synchronized (thread safe); otherwise, false.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
```

Comment:
The member name is `System.Collections.ICollection.IsSynchronized`, not `Rank`.

Reply:
I think GitHub might have a bug, because I'm seeing intermixed lines that do not belong there. You showed me in the other PR that the content was correct.